.env.production

```
@@ -9,3 +9,4 @@ VITE_FEATURE_CELO=false
 
 # mixpanel
 VITE_MIXPANEL_TOKEN=9d304465fc72224aead9e027e7c24356
+VITE_FEATURE_STARGATE_SWAP=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_FEATURE_STARGATE_SWAP key should go before the VITE_MIXPANEL_TOKEN key

(UnorderedKey)